src/components/List/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Card from '../Card/index.js';
+import './list.css';
+
+export defualt class List extends React.Component() {
+    constructor(props){
```

Comment:
you can actually delete L6-10, since it's doing exactly what the `super()` is doing